src/core/Clock.js

```diff
@@ -14,9 +18,9 @@ function Clock( autoStart ) {
 
 }
 
-Object.assign( Clock.prototype, {
+Clock.prototype = {
```

Comment:
This format is needed for the type system to automatically pick up the types of the class properties and methods without having to declare separate type definitions using `/** @typedef {...} */`.